jenkins-blood.planx-pla.net/manifest.json

```diff
@@ -314,6 +315,116 @@
     "config_index": "jenkins_configs_alias",
     "auth_filter_field": "auth_resource_path"
   },
+  "mariner": {
+    "storage": {
+      "s3": {
+        "name": "workflow-engine-garvin",
+        "region": "us-east-1"
+      }
+    },
+    "secrets": {
+      "awsusercreds": {
+        "name": "workflow-bot-g3auto",
+        "key": "awsusercreds.json"
+      }
+    },
+    "jobs": {
+      "engine": {
+        "labels": {
+          "app": "mariner-engine"
+        },
+        "serviceaccount": "mariner-service-account",
```

Comment:
Shouldn't we use prefixed service accounts in case we need to change things between preprod and prod?